tests/test_request_stream.py

```diff
@@ -83,7 +83,7 @@ def test_request_stream_app():
                 body = await request.stream.get()
                 if body is None:
                     break
-                response.write(body.decode('utf-8'))
+                await response.write(body.decode('utf-8'))
```

Comment:
PR looks good. Hoever, won't this require an update to the docs to use the `await` syntax for streaming?

Reply:
Yes. You're right.
I changed the response-streaming example, but I forgot to change the response-streaming docs.